src/server/database/getRethinkConfig.js

```diff
@@ -0,0 +1,34 @@
+import {readCert} from './readCert';
+import flag from 'node-env-flag';
+
+export const getRethinkConfig = () => {
+
+  let config = {
+    host: process.env.DATABASE_HOST || 'http://rethinkdb.dev.azk.io' || 'localhost',
```

Comment:
I only use this to quickly switch between working within azk.io and outside of it, once I'm done, I'll be removing it, perhaps for better clarity I'll use // instead of || :)